src/listeners/interactionCreate.ts

```diff
@@ -88,6 +88,8 @@ client.on("interactionCreate", async interaction => {
 		setTimeout(() => timestamps.delete(interaction.user.id), cooldownAmount)
 	}
 
+	const langFromLocale = author.lang ?? transformDiscordLocale(interaction.locale)
```

Comment:
On today's episode of this is not how you javascript: this needs to be declared at the very top of the file along with the other constants (aka before getString is used)